src/EODReminder.js

```diff
@@ -15,12 +15,13 @@ const fs = require('fs');
 const path = require('path');
 
 const { execSync } = require('child_process');
-const { WebClient } = require('@slack/client');
-const cpCommand = 'cp ../config-files/RAs.txt ../config-files/sleepyRAs.txt';
+const { WebClient } = require('@slack/web-api'); //const { WebClient } = require('@slack/client');
+
+const cpCommand = 'cp ' + path.join(__dirname, '../config-files/RAs.txt') + " " + path.join(__dirname, '../config-files/sleepyRAs.txt') //'cp ../config-files/RAs.txt ../config-files/sleepyRAs.txt';
 const slackToken = fs.readFileSync(path.resolve(__dirname, '../config-files/SLACK_TOKEN'), 'utf8');
 const web = new WebClient(slackToken);
-const channelIDs = require('../config-files/channelID.json');
-const eodReminders = require('../config-files/EODReminderTimes.json');
+const channelIDs = require(path.join(__dirname, '../config-files/channelID.json'));
+const eodReminders = require(path.join(__dirname, '../config-files/EODReminderTimes.json'));
```

Comment:
I don't think `path.join` is needed here. `require` should handle the relative paths.

Reply:
However, I need the file to run using absolute paths for testing purposes. Without dirname, it'll only work if you run the file within the same directory.